src/main/java/org/apache/commons/io/output/AppendableWriter.java

```diff
@@ -124,10 +126,7 @@ public T getAppendable() {
      */
     @Override
     public void write(final char[] cbuf, final int off, final int len) throws IOException {
-        Objects.requireNonNull(cbuf, "cbuf");
-        if (len < 0 || off + len > cbuf.length) {
-            throw new IndexOutOfBoundsException("Array Size=" + cbuf.length + ", offset=" + off + ", length=" + len);
-        }
+        IOUtils.checkFromIndexSize(cbuf, off, len);
```

Comment:
Javadoc `@throws IndexOutOfBoundsException ...` is missing.

Reply:
Fixed in https://github.com/apache/commons-io/pull/790/commits/8d520f72c3698440e2275b860a991a41757ad2fb